### DISCUSSION TITLE
part-ii/2018-88/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#233] [Question comments]
Part II 2018 Logic and Set Theory: Paper 2, Section II, G 

https://questions.tripos.org/part-ii/2018-88/

**Comment by artie2000** [comment]
<details>
<summary>Poset isn't complete:</summary>

Consider $\sup P$, and show that there's a greater element.

</details>
<details>
<summary>Order-preserving function with no fixed point:</summary>

Well-order $\mathbb{R}$, and let $f(A)$ be the least element of $\mathbb{R}\setminus A$.  Then map $A\to A\cup\{f(A)\}$ (check it's order preserving).

</details>

**Comment by dropdbassett** [comment]
An alternative for each:
<details><summary>Poset isn't complete</summary>

Note the sets $\mathbb{Q} + x$ for $x \in [0, 1)$ are countable, but any upper bound contains their union, which is all of $\mathbb{R}$, and hence isn't countable. So they have no upper bound in $P$.
</details>

<details><summary>Order-preserving function with no fixed point</summary>

Well-order the reals, then $A \mapsto \{x^+ : x \in A\}$. This is a bijection between the original set and its image, and the least element strictly increases, so we never have $A \mapsto A$.
</details>

**Reply by artie2000** [reply]
Oop you just made me realise my solution doesn't work (fixed)
